contrib/bump-version.sh

```diff
@@ -21,3 +21,9 @@ fi
 
 $SED -i "/^#define VERSION .*/c\#define VERSION \"$version\"" src/Common.h
 $SED -i "/^Version: .*/c\Version: $version" contrib/rpm/fulcrum.spec
+
+
+#man page
+_date=$(date '+%B %d, %Y')
+$SED -i "1s/\%.*/\% FULCRUM\(1\) Version $version \| Fulcrum Manual/" doc/unix-man-page.md
```

Comment:
It would be nice with a better sed command that only replaces the version number